strawberryfields/gbs/similarity.py

```diff
@@ -201,3 +212,155 @@ def event_to_sample(photon_number: int, max_count_per_mode: int, modes: int) ->
             available_modes.remove(j)
 
     return sample
+
+
+def fac_prod(orbit: list) -> int:

```

Comment:
🤔 Given that we only use this function twice, and that it can also be done very simply as
```python
from scipy.special import factorial
import numpy as np

l = [1, 2, 3]
np.prod(factorial(l))
```
I think we should remove this function, both to lighten the tests and also to not reinvent the wheel.

Reply:
Ok, so I eventually changed this function to a single underscore function, but after talking with @josh146 @ixfoduap we are going to delete this function and just have it directly in the code.